src/main/java/aquality/selenium/browser/devtools/EmulationTools.java

```diff
@@ -0,0 +1,34 @@
+package aquality.selenium.browser.devtools;
+
+import aquality.selenium.browser.AqualityServices;
+import aquality.selenium.core.localization.ILocalizedLogger;
+import org.openqa.selenium.devtools.DevTools;
+import org.openqa.selenium.devtools.v96.emulation.Emulation;
+
+import java.util.Optional;
+
+public class EmulationTools {
+
+    private final DevTools tools;
+    private final ILocalizedLogger localizedLogger = AqualityServices.getLocalizedLogger();
+
+    public EmulationTools(DevTools tools) {
+        this.tools = tools;
+    }
+
+
+    public void setGeolocationOverride(double lat, double lng, int accuracy) {
+        localizedLogger.info("loc.browser.emulation.geolocation.set");
+        tools.send(Emulation.setGeolocationOverride(Optional.of(lat), Optional.of(lng), Optional.of(accuracy)));
```

Comment:
Any ideas why these parameters are optional in Selenium? Is it possible to override latitude but leave longitude the same, passing Optional.empty() ? If it is, then we need to give such ability to user - by creation of another overload with optional parameters